File: CartQuestAndroid/app/src/main/java/com/amoghghadge/cartquestandroid/data/model/KrogerStore.kt
package com.amoghghadge.cartquestandroid.data.model

data class KrogerStore(
    val locationId: String,
    val chain: String,
    val name: String,
    val address: KrogerAddress,
    val geolocation: KrogerGeolocation
)

data class KrogerAddress(
    val addressLine1: String,
    val city: String,
    val state: String,
    val zipCode: String
)

data class KrogerGeolocation(
    val latitude: Double,
    val longitude: Double
)

File: CartQuestiOS/CartQuestiOS/Models/KrogerStore.swift
import Foundation

struct KrogerStore: Codable, Identifiable {
    var id: String { locationId }
    var locationId: String
    var chain: String
    var name: String
    var address: KrogerAddress
    var geolocation: KrogerGeolocation
}

struct KrogerAddress: Codable {
    var addressLine1: String
    var city: String
    var state: String
    var zipCode: String
}

struct KrogerGeolocation: Codable {
    var latitude: Double
    var longitude: Double
}
